locales/templates/app.pot

```diff
@@ -70,13 +70,13 @@ msgctxt "text_selection_search_action"
 msgid "Search in Firefox Rocket"
 msgstr ""
 
-#. Content of about Rocket. Argument 1 is the app name of the content. i.e.x.
-#. Firefox Rocket
+#. Content of about Rocket.     %1$s is the app name of the content. i.e.x.
+#. Firefox Rocket     %2$s is "Mozilla"
```

Comment:
Nit: "content of about" --> "content about". Also there seems to be a lot of spaces before %1$s and  %2$s, not sure that's expected

Reply:
The sentence is parsed as Content of "about Rocket." I tried to add that double quote here but a2po isn't buying it and skips the double quote.